README.md

```diff
@@ -44,37 +65,38 @@ Plone offers fine-grained permission control over content and actions.
 Plone is easy to set up, extremely flexible,
 and provides you with a system for managing web content that is ideal for project groups, communities, websites, extranets, and intranets.
 
-- *Plone is easy to install.*
+- **Plone is easy to install.**
   Several installation options are available for either your local machine or on servers in the cloud.
 
-- *Plone is easy to use.*
+- **Plone is easy to use.**
   The Plone Team includes usability experts who have made Plone easy and attractive for content managers to add, update, and maintain content.
 
-- *Plone is international.*
+- **Plone is international.**
   The Plone interface has more than 35 translations, and tools exist for managing multilingual content.
 
-- *Plone is standard.*
+- **Plone is standard.**
   Plone carefully follows standards for usability and accessibility.
   Plone pages are compliant with US Section 508, and the W3C's AAA rating for accessibility.
 
-- *Plone is Open Source.*
+- **Plone is Open Source.**
   Plone is licensed under the GNU General Public License, the same license used by Linux.
   This gives you the right to use Plone without a license fee, and to improve upon the product.
 
-- *Plone is supported.*
+- **Plone is supported.**
   There are over three hundred developers in the Plone Development Team around the world, and a multitude of companies that specialize in Plone development and support.
```

Comment:
Is that still correct, or has the number grown? I'm not sure how you would count "developers". Perhaps this? https://github.com/orgs/plone/people

Reply:
I have queried github api this afternoon.
I count commits from more than 300 devs since the beginning of 2022 in the plone organization.

Reply:
Our three developer stars are @mauritsvanrees @sneridagh and @jensens 😉

Reply:
More precisely:
146 committers in 2021 
141 in 2022.
For a total of 196 committers.
96 among them committed both years.

Reply:
Is that the GitHub Plone organization only, or does it include collective and zope?

Reply:
@stevepiercy For now `plone` only. I am improving my script before running it on zope and collective.